corehq/apps/cloudcare/templates/formplayer/dependencies.html

```diff
@@ -35,6 +35,32 @@
   <script src="{% static 'clipboard/dist/clipboard.js' %}"></script>
   <script src="{% static 'reports/js/readable_form.js' %}"></script>
   <script src="{% static 'url-polyfill/url-polyfill.js' %}"></script>
+  <script>
+      function getCookie(cname) {
```

Comment:
was this function copied from somewhere? It seems like it could use a reference or a test.

Do we not have any utilities that can do this for us in jquery or somewhere?

Reply:
Yeah, I coped it from here https://www.w3schools.com/js/js_cookies.asp under the heading `A Function to Get a Cookie`

> Do we not have any utilities that can do this for us in jquery or somewhere?

I'm not sure about this.

Reply:
We use a fork of jquery cookie: https://github.com/dimagi/jquery-cookie

Reply:
Thanks for the link @orangejenny. I looked at the repo and it says `No longer maintained, superseded by JS Cookie:` Is it still okay to use that library?

Reply:
Yeah, it's still okay to use, for the sake of being consistent with existing code. But there's discussion happening in the #dependency-pod channel on slack about replacing all of our jquery cookie usage with js-cookie.